<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsApplet</class>
 <widget class="QWidget" name="SettingsApplet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>640</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="addressLabel">
       <property name="text">
        <string>MongoDB Address:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="address_input"/>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="save_button">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="portLabel">
       <property name="text">
        <string>MongoDB Port:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="port_input"/>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>406</width>
         <height>406</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>406</width>
         <height>406</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>logo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>WebUI on localhost:5000</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="startWebUIButton">
           <property name="text">
            <string>Start WebUI</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stopWebUIButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Stop WebUI</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
